rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
+    @person = person
+    person.rentals << self
```

Comment:
- [ ] Kindly add the `rentals` property in the `Person` class as `attr_accessor` so you can add value to it inside the `initialize` in the `Rental` class.